pkg/hostagent/port_darwin.go

```diff
@@ -149,8 +151,25 @@ func (plf *pseudoLoopbackForwarder) forward(ac *net.TCPConn) error {
 		return err
 	}
 	defer unixConn.Close()
-	bicopy.Bicopy(ac, unixConn, nil)
-	return nil
+
+	g, _ := errgroup.WithContext(context.Background())
+
+	g.Go(func() error {
+		_, err := io.Copy(unixConn, ac)
+		if errors.Is(err, io.EOF) {
+			err = nil
```

Comment:
The previous code does not handle io.EOF, so this implementation may behave differently.

Reply:
Indeed, this is matching the behavior here: https://github.com/lima-vm/lima/blob/ca778e338ab95524555ba4d23bd6398be5a6ee0f/pkg/portfwd/client.go#L30-L43

However that behavior appears to be nonsense given the documentation of [`io.Copy`](https://pkg.go.dev/io?utm_source=godoc#Copy):
> A successful Copy returns err == nil, not err == EOF. Because Copy is defined to read from src until EOF, it does not treat an EOF from Read as an error to be reported.

So I've removed that and mentioned in the commit message.